PWGJE/Tasks/PhotonIsolationQA.cxx

```diff
@@ -302,23 +301,23 @@ struct PhotonIsolationQA {
         MC_Info.fill(HIST("hRho_Perpen_Cone"), Rho_Perpen_Cone);
         MC_Info.fill(HIST("hShowerShape"), mccluster.m02(), mccluster.m20());
         MC_Info.fill(HIST("hSigmaLongvsPtIso"), mccluster.m02(), Pt_iso);
+        MC_Info.fill(HIST("hE_M02_M20_NLM_NCells_PtIso"), mccluster.energy(), mccluster.m02(), mccluster.m20(), mccluster.nlm(), mccluster.nCells(), Pt_iso);
         fillABCDHisto(MC_Info, mccluster, Pt_iso);
-
         // acces mc true info
         auto ClusterParticles = mccluster.mcParticle_as<aod::StoredMcParticles_001>();
         bool background = true;
         for (auto& clusterparticle : ClusterParticles) {
           if (clusterparticle.pdgCode() == 22) {
             MC_Info.fill(HIST("hMcParticlesToCluster"), clusterparticle.e());
+            MC_Info.fill(HIST("hMcParticleProcessCode"), clusterparticle.getProcess());
             MC_Info.fill(HIST("hMcParticleStatusCode"), clusterparticle.getGenStatusCode());
-            if ((clusterparticle.getGenStatusCode() >= 21) && (clusterparticle.getGenStatusCode() <= 29))
+            if ((clusterparticle.getProcess() >= 21) && (clusterparticle.getProcess() <= 29))
```

Comment:
i am not completely sure how status codes work in O2, but please make sure that both fragmentation photons AND prompt photons get accepted as signal. In addition, signal photons should also fulfill and isolation on GENERATOR level